docs/source/architecture/service_fees.rst

```diff
@@ -3,89 +3,67 @@
 
 **Maximum fee rate**
 
-X,XXX GWEI (~$X.XX x10^-X) *per period*
-
-XXX,XXX GWEI ($X.XXX) *per year*
+500 GWEI (~$1.59 x10^-4) *per period*
 
 **Default fee rate**
 
-XXX GWEI (~$X.XX x10^-X) *per period*
-
-XX,XXX GWEI ($X.XXXX) *per year*
+50 GWEI (~$1.59 x10^-5) *per period*
 
-*1 GWEI = 10^-9 ETH*
+*USD conversion uses the 100-day rolling average of the ETHUSD exchange rate of ~$318, on October 14th 2020. 1 GWEI = 10^-9 ETH.*
 
-*USD conversion utilizes the ETHUSD 100-day rolling average.*
+The minimum and maximum fee rate are a lower and upper bound to constrain the fee rate a staker may offer. The default fee rate is the rate that will be displayed and offered to Alices if the staker chooses not to configure this parameter themselves, or chooses a rate outside the boundaries of the global fee range. The default rate will also be used if the range's boundaries are updated, a staker's specified rate *now* falls outside the range, and they fail to change it.
 
-Note that the minimum and maximum fee rate are a lower and upper bound to constrain the fee rate a staker may offer. The default fee rate is the rate that will be displayed and offered to Alices if the staker chooses not to configure this parameter themselves, or chooses a rate outside the boundaries of the global fee range. The default rate will also be used if the range's boundaries are updated, a staker's specified rate *now* falls outside the range, and they fail to change it.
-
-The fee range must be adhered to in identical fashion by all NuCypher stakers, regardless of their stake size or capacity. The fee range applies to all sharing policies, irrespective of the volume of re-encryption requests or other distinguishing attributes. It also applies equally to all periods in the future, until the moment that the global fee range’s parameters are adjusted or the range is removed, via official governance processes (see below). If an update of this sort occurs, sharing policies that were previously established, but have not yet expired, should not have the per-period fee rate retroactively modified.
-Note that the global fee range is only applicable to stakers and Ursulas. Alices are free to pay as high a rate as they like.
+The fee range must be adhered to in identical fashion by all NuCypher stakers, regardless of their stake size or capacity. The fee range applies to all sharing policies, irrespective of the volume of re-encryption requests or other distinguishing attributes. It also applies equally to all periods in the future, until the moment that the global fee range’s parameters are adjusted or the range is removed, via official governance processes (see below). If an update of this sort occurs, sharing policies that were previously established, but have not yet expired, should not have the per-period fee rate retroactively modified. Note that the global fee range is only applicable to stakers and Ursulas. Alices are free to pay as high a rate as they like.
```

Comment:
This is an extraordinary important paragraph, that adds tons of very important details that I actually never considered. Great work!